python/cudf/cudf/tests/test_dataframe.py

```diff
@@ -3064,12 +3082,21 @@ def test_all(data):
         ),
     ],
 )
-def test_any(data):
+@pytest.mark.parametrize("axis", [0, 1])
+def test_any(data, axis):
     # Pandas treats `None` in object type columns as True for some reason, so
     # replacing with `False`
     if np.array(data).ndim <= 1:
         pdata = pd.Series(data).replace([None], False)
         gdata = Series.from_pandas(pdata)
+
+        if axis == 1:
+            with pytest.raises(NotImplementedError):
```

Comment:
Why are we always raising NotImplementedError with axis=1 for any?

Reply:
This is because there is no column-wise parsing(`axis=1`) in series, for example..

```python
>>> import pandas as pd
>>> s = pd.Series([1,2,3])
>>> s.all()
True
>>> s.all(axis=1)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/conda/envs/cudf/lib/python3.7/site-packages/pandas/core/generic.py", line 11723, in logical_func
    filter_type="bool",
  File "/conda/envs/cudf/lib/python3.7/site-packages/pandas/core/series.py", line 4066, in _reduce
    self._get_axis_number(axis)
  File "/conda/envs/cudf/lib/python3.7/site-packages/pandas/core/generic.py", line 411, in _get_axis_number
    raise ValueError("No axis named {0} for object type {1}".format(axis, cls))
ValueError: No axis named 1 for object type <class 'pandas.core.series.Series'>
>>> import cudf
>>> s = cudf.from_pandas(s)
>>> s.all(axis=0)
True
>>> s.all(axis=1)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/conda/envs/cudf/lib/python3.7/site-packages/cudf/core/series.py", line 1271, in all
    raise NotImplementedError("axis parameter is not implemented yet")
NotImplementedError: axis parameter is not implemented yet
>>> 
```